### PR TITLE
Update Sign Up examples to use initialState="signUp"

### PR DESCRIPTION
**Pull request by ericclemmons**
*Description of changes:*

- [x] Instead of `authMachine`, use `createAuthenticatorMachine({ initialState?, loginMechanisms? })`
- [x] Fix `SignInSignUpTabs.tsx` to be controlled
- [x] Update Angular
- [x] Update Vue 
- [x] Update `sign-up-*` tests to use `initialState` instead of clicking the tab

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by ericclemmons** [comment]
@wlee221 @ErikCH I could use your help figuring out the types for these changes. It _looked_ right, but clearly not :D

**Comment by wlee221** [comment]
Ah the input binding should be 

`[initialState]="'signUp'"`, or just `initialState="signUp"`.

Putting brackets around the `input` signals that the value would be a javascript value, not a string html literal. And the surrounding double quote `" value "` wraps that javascript value (think of it as React `{ myVariable }`).

File: packages/vue/src/components/authenticator.vue [hunk 1]
```diff
@@ -300,8 +298,11 @@ onBeforeMount(() => {
 
 const attrs = useAttrs();
 
-const { loginMechanisms } = withDefaults(
-  defineProps<{ loginMechanisms?: LoginMechanism[] }>(),
+const { initialState, loginMechanisms } = withDefaults(
+  defineProps<{
+    initialState: AuthenticatorMachineOptions['initialState'];
+    loginMechanisms: AuthenticatorMachineOptions['loginMechanisms'];
```
Comment (ErikCH) [inline packages/vue/src/components/authenticator.vue:304]: These should be optional

```suggestion
    initialState?: AuthenticatorMachineOptions['initialState'];
    loginMechanisms?: AuthenticatorMachineOptions['loginMechanisms'];
```

**Review by ErikCH — APPROVED** [review]
👍

**Pull request merged**